.travis.yml

```diff
@@ -21,13 +28,18 @@ cache:
   - $HOME/.gradle/caches/
   - $HOME/.gradle/wrapper/
 
-before_install: echo -e "org.gradle.parallel = true\norg.gradle.jvmargs=-Xms256m -Xmx1024m\norg.gradle.configureondemand=true\ntasks.withType(Test) {\n maxParallelForks = Runtime.runtime.availableProcessors()\n}" > gradle.properties
+before_install:
+  - if [[ "${JAVA_HOME}" == *java-7-openjdk* ]]; then
+        sudo apt-get update && sudo apt-get install openjdk-7-jdk;
+        jdk_switcher use openjdk7;
+        export _JAVA_OPTIONS="-Xmx2048m -Xms512m -Djava.security.properties==${PWD}/java.security-openjdk-7";
```

Comment:
I can't use `gradle.properties` to setup `_JAVA_OPTIONS`. I had to resort to overwriting this variable instead.